sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -750,19 +752,19 @@ public HttpPipeline getHttpPipeline() {
      * @return A {@link PagedFlux} of created models and the http response.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedFlux<ModelData> createModels(List<String> models) {
+    public PagedFlux<DigitalTwinModelData> createModels(List<String> models) {
```

Comment:
We call this `ModelData` in the .NET client, maybe call it the same here as well?

Reply:
sure, we can reuse the same name now that they are in 2 different packages.